token_server/token_server.js

```diff
@@ -4,6 +4,7 @@ import {createServer} from 'http';
 import parseBody from 'urlencoded-body-parser';
 import {storeGoogleCredentials} from './aws_get_credentials.js';
 import {generateEarthEngineToken} from './ee_token_creator.js';
+import {TOKEN_SERVER_ORIGINS} from '../docs/owner.js';
```

Comment:
I think deploying token server doesn't work with local files outside this directory, sadly :(

Reply:
Sad, moved to a file in this dir